### ISSUE TITLE
Option 'bt-metadata-only' not work when torrent file existed.

### ISSUE THREAD

**Issue by coolerfall** [aria2/aria2#2084]
I tried to download torrent file with option `bt-metadata-only`, it works the first time. But when I added the same magnet uri again, it downloaded the whole files but not metada.

**Comment by houssem-108** [comment]
did you also include `--bt-save-metadata` option ? 
`--bt-metadata-only` on it's own doesn't create a torrent file, and shouldn't download files either, afaik.

**Comment by coolerfall** [comment]
Yes, with  `--bt-save-metadata` option. But it download files when the same magnet uri added again. 
This is the operation:  `add magnet uri`-->`only torrent file downloaded`-->`the task completed`-->`add same magnet uri again`-->`the whole file will be downloaded`
